src/lsst/cm/tools/core/panda_utils.py

```diff
@@ -179,42 +179,45 @@ def determine_error_handling(dbi: DbInterface, errors_agg: dict, max_pct_failed:
     decision_results = []
     for key in errors_agg.keys():
         # for a given error, try to make a match
-        error_item = errors_agg[key]
+        error_items = errors_agg[key]
```

Comment:
So. `error_items` is just a way to loop through each individual error and access what we know about it?

Reply:
Yep. Since error_items doesn't necessarily return a single value for a given key, we get back a list and then just iterate over it.